src/Microsoft.Diagnostics.NETCore.Client/DiagnosticsClient/EventPipeSession.cs

```diff
@@ -13,6 +13,17 @@ namespace Microsoft.Diagnostics.NETCore.Client
 {
     public class EventPipeSession : IDisposable
     {
+        //! This is CoreCLR specific keywords for native ETW events (ending up in event pipe).
+        //! The keywords below seems to correspond to:
+        //!  GCKeyword                          (0x00000001)
+        //!  LoaderKeyword                      (0x00000008)
+        //!  JitKeyword                         (0x00000010)
+        //!  NgenKeyword                        (0x00000020)
+        //!  unused_keyword                     (0x00000100)
+        //!  JittedMethodILToNativeMapKeyword   (0x00020000)
+        //!  ThreadTransferKeyword              (0x80000000)
+        public const long DefaultRundownKeyword = 0x80020139;
```

Comment:
Do we want this to be a public constant? It might be better to make this an internal implementation detail so that other code doesn't depend on this exact value. I also think that this should be lifted into M.D.Monitoring.EventPipe and dotnet-trace.

Reply:
I kind of wanted it to stay public, this is 
a.) not a secret
b.) people should think about it when they request rundown - do you really need all these rundown events, or think about the consequence when their tools are run on top of older runtimes,
c.) allow people to construct profiles with it. (Oh, I want the default rundown but add this or remove that), and
d.) avoid code duplication

Reply:
a) Never said it was a secret. I'd argue that the default rundown keyword set is an implementation detail for each version of .NET. What happens when it is decided that the default rundown keyword set needs to be changed in a newer .NET version?
b) Up until now, a caller only had the ability to say "give me rundown" or "don't give me rundown". That's probably what most are going to do regardless of providing the ability to customize it. My guess is that the rundown keyword customization will only be used by the code in these libraries to construct these profiles, unless you have customers specifically asking to customize the rundown data.
c) If customers want to describe "I want default rundown, but add X and remove Y", then that should probably be described with a bitmask and additions (e.g. `addition = X, mask = ~Y`).
d) We avoid duplication by moving the `dotnet-*` tools to use the M.D.M.EventPipe library, but that's outside the scope of this PR.

Once you make this a public constant, you can't remove it and you can't change its value without declaring breaking changes. I would make it internal and wait for feedback that callers want the value to be public with a good justification.

Reply:
>I'd argue that the default rundown keyword set is an implementation detail for each version of .NET. What happens when it is decided that the default rundown keyword set needs to be changed in a newer .NET version?

That's a great insight, I haven't considered that possibility. I will make it internal for now.